### ISSUE TITLE
Add a warning when changing password in TrueCrypt mode

### ISSUE THREAD

**Issue by TomasRiker** [veracrypt/VeraCrypt#1015]
Today I changed the password of a container that I had always been mounting in TrueCrypt mode. When changing the password, I ticked the "TrueCrypt mode" checkbox. The password was changed, and I was no longer able to mount the container in TrueCrypt mode (error 9170). Seems like the container got "upgraded" to a regular VeraCrypt container in the process. I didn't see any warning or hint about that. I already thought I had lost access to my container. Only by accident I tried to mount it without TrueCrypt mode, and it worked.
Bottom line: There should be some prominent hint or warning directly in the UI when changing the password in TrueCrypt mode.
(I wasn't using the very latest VeraCrypt version when I changed the password, so if a warning has been added already, my apologies!)
